rust/ddlog_benches/README.md

```diff
@@ -0,0 +1,12 @@
+# ddlog_benches
+
+DDlog has a unique benchmarking scenario since it's a compiler, so benchmarking the programs
+it generates is a little more involved than normal, which is why we use [`cargo-make`] to automate
+the setup, build and bench process in a cross-platform way. To run the benchmarks in this crate,
+first install `cargo-make` with the following command (`--force` makes cargo install the latest version)
```

Comment:
How do you run the benchmark after installing `cargo-make`?

Reply:
`cargo make benchmark-all` at the moment, I'll probably add settings later so you can choose which benchmarks and so on, but there's no point in that now. I'll add the command to the readme, thanks for reminding me!